packages/env/README.md

```diff
@@ -297,6 +297,24 @@ Positionals:
 
 ### `wp-env run [container] [command]`
 
+The run command can be used to open shell sessions or invoke WP-CLI commands.
+
+<div class="callout callout-alert">
+To run a WP-CLI command that includes optional arguments, enclose the command in quotation marks; otherwise, the arguments are ignored.
```

Comment:
Small suggestion to include an explainer for why this happens:

```suggestion
To run a WP-CLI command that includes optional arguments, enclose the command in quotation marks; otherwise, the arguments are ignored. This is because flags are normally passed to `wp-env` itself, meaning they are aren't considered part of the argument which specifies the command. With quotation marks, `wp-env` considers everything inside quotation marks the "command" argument.
```

Reply:
ok